starter_code/index.js

```diff
@@ -1 +1,18 @@
 const Elevator = require('./elevator.js');
+
+const Person = require('./person.js');
+
+const newElevator = new Elevator();
+
+const jennie = new Person('Jennie', 3, 9);
```

Comment:
**const** is a signal that the identifier won’t be reassigned.
**let** is a signal that the variable may be reassigned, such as a counter in a loop, or a value swap in an algorithm.
you should use let in these ones as these are not really constant.